mobile/src/screens/ConfirmDevice.tsx

```diff
@@ -163,6 +100,8 @@ export default function ConfirmDevice({ navigation }: ConfirmDeviceProps) {
           }
         }, 200);
       } else {
+        const oid = await SecureStore.getItemAsync("oid");
```

Comment:
As specified in other classes. Can we rather use an enum for the secure store keys instead of the strings directly?